### PR TITLE
added docker documentation

### PR DESCRIPTION
**Pull request by sukhpreet-singh1**
I have added the docker documentation for running application ,and tried it on my own system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,12 +113,12 @@ These are the prerequisites required to run this application.
   npm start
 ```
 
-4. To run the application using Docker in server
-we first build app container using 
+4. To run the application using Docker in server  
+   we first build app container using 
 ```
   docker build -t getting-started
```
[the remark this answers is not in the document]
Comment (sukhpreet-singh1) [reply README.md:119]: its already mentioned that directory and file location is server folder

[the remark this answers is not in the document]
Comment (sukhpreet-singh1) [reply README.md:119]: getting-started name is given because its the starting phase

**Comment by mn-gt** [comment]
why docker-documentation is not in Readme file or it is somewhere else?
why even after PR submission, feature is not being updated in main [site](http://www.manthan-app.org/) so that others can create further issues?

**Comment by Vidisha-kayal** [comment]
@sukhpreet-singh1 please respond to the requested changes else the issue will be assigned to someone else

**Comment by aavishkarmishra** [comment]
> why docker-documentation is not in Readme file or it is somewhere else?
> why even after PR submission, feature is not being updated in main [site](http://www.manthan-app.org/) so that others can create further issues?

This pull request is not merged, you can see changes after merging.

**Pull request closed without merging**